src/components/ContactsList/CategorizedList.jsx

```diff
@@ -12,14 +13,14 @@ const CategorizedList = ({ contacts }) => {
   const categorizedContacts = categorizeContacts(contacts, t('empty-list'))
 
   return (
-    <ol className="list-contact">
+    <Table className="u-nolist">
```

Comment:
I thought that u-nolist would be used on list elements like (ul, ol) to remove list item bullets, why is it necessary to add it on a table.